Documentation/ceph-csi-drivers.md

```diff
@@ -202,12 +193,22 @@ mgr.a
 	caps: [mon] allow *
 	caps: [osd] allow *
 
-#encode admin/user key
-sh-4.2# echo -n AQD0pK1cqcBDCBAAdXNXfgAambPz5qWpsq0Mmw==|base64
-QVFEMHBLMWNxY0JEQ0JBQWRYTlhmZ0FhbWJQejVxV3BzcTBNbXc9PQ==
-#or
-sh-4.2# ceph auth get-key client.admin|base64
-QVFEMHBLMWNxY0JEQ0JBQWRYTlhmZ0FhbWJQejVxV3BzcTBNbXc9PQ==
+#get  user key
+sh-4.2# ceph auth get-key client.admin
+AQD0pK1cqcBDCBAAdXNXfgAambPz5qWpsq0Mmw==
+```
+
+Example:
+
+```console
+apiVersion: v1
+kind: Secret
```

Comment:
The csi driver will only be able to load this secret if it's in the same namespace. What if there are multiple rook-ceph clusters? Will the csi driver know which secret to load? Seems like we need a similar approach to the mon endpoint configmap where multiple clusters are supported.

Reply:
the secret name and the namespace will be part of the storage class created, I think we should not worry about it.

Reply:
@travisn more specifically the secret and its namespace are provided in the StorageClass, kubernetes ensures that calls that need the said secret are fetched from the said namespace and passed to the CSI plugins, hence the secret's namespace can be anything, as kubernetes takes care of the the same.